adapters/repos/db/aggregator/grouper.go

```diff
@@ -70,7 +71,7 @@ func (g *grouper) groupAll(ctx context.Context) ([]group, error) {
 func (g *grouper) groupFiltered(ctx context.Context) ([]group, error) {
 	s := g.getSchema.GetSchemaSkipAuth()
 	ids, err := inverted.NewSearcher(g.db, s, g.invertedRowCache, nil).
-		DocIDs(ctx, g.params.Filters, false, g.params.ClassName)
+		DocIDs(ctx, g.params.Filters, traverser.UnderscoreProperties{}, g.params.ClassName)
```

Comment:
We're now passing three arguments to `DocIDs()` which are all also contained in `traverser.GetParams`: `*filters.LocalFilter`, `traverser.UnderscoreProperties` and the class name.

Could we instead always pass a `traverser.GetParams` and therefore make the Signature of `DocIDs()` (and I assume also `Objects()` simpler?

For this change to really make sense, we'd probably also have to reflect it in the methods calling the searchers, which I think is mostly `*Index.objectSearch()` and `*Shard.objectSearch()`, but I think they could be adjusted as well?

Reply:
Outscoped to #1294